lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
What I mean is, when using `raygun.send(err)`, the `throw err` line within the `.then()` gets swallowed.  Everything continues on as normal, no error is ever seen, as if it's not even there.  When using Bluebird via `Promise.resolve(raygun.send(err))`, everything works as expected, and the error gets propagated up the chain